package.json

```diff
@@ -57,7 +57,7 @@
     "valid-url": "=1.0.9",
     "validator": "=7.0.0",
     "validator.js": "=2.0.3",
-    "wamp-socket-cluster": "LiskHQ/wamp-socket-cluster#47b53a59f72441686314296f619b67cc8e855e21",
+    "wamp-socket-cluster": "LiskHQ/wamp-socket-cluster#development",
```

Comment:
we are preparing 2.0.0 wamp-socket-cluster release, but it is still not ready. Changes in this PR are relaying in some changes which are present on development branch though.